### ISSUE TITLE
Cannot override -Xmx and -Xms with jvm.args

### ISSUE THREAD

**Issue by jtheuer** [boxfuse/boxfuse-issues#127]
Hi,

I start my application on aws with jvm.args given on the command line.

```
boxfuse run myapp:<version> -env=production -jvm.args=-Xmx4g -configfile=boxfuse-production.conf -elb=myapp-production -X -envvars.SPRING_PROFILES_ACTIVE=live,production
[...]
14:16:05.642 healthcheck.path -> /api/ping
14:16:05.642 apptype -> load-balanced
14:16:05.642 subnets -> [...]
14:16:05.643 app -> myapp
14:16:05.643 Skipping non-existent config file: [...]
14:16:05.643 Loading configuration from [...] ...
14:16:05.643 capacity -> 1-3:r3.2xlarge:cpu30-70
14:16:05.643 instanceprofile -> arn:aws:iam::[...]
14:16:05.643 jvm.args -> -Xmx4g
```

I see in console log this output:

```
BOXFUSE_JVM_ARGS=-Xmx4g
BOXFUSE_JVM_JMX=false
BOXFUSE_JVM_MAIN_CLASS=org.springframework.boot.loader.JarLauncher
[...]
******************************************************************************

Launching App ...
/jre/bin/java   -Xmx51980m -d64 -Xms10m -XX:OnOutOfMemoryError="killall -3 java" -Djavax.net.ssl.trustStore=/cacerts/cacerts -Djava.awt.headless=true -Djava.security.egd=file:/dev/./urandom -Djdk.reflect.allowGetCallerClass=true    '-[...] '-DBOXFUSE_JVM_ARGS=-Xmx4g' '-DBOXFUSE_JVM_JMX=false' '-DBOXFUSE_JVM_MAIN_CLASS=org.springframework.boot.loader.JarLauncher' '- [...]  -Xmx4g -cp "/app/wanderwalter-server-2.117-SNAPSHOT-fat.jar" org.springframework.boot.loader.JarLauncher 
```

Hope I didn't strip out too much but essentially **my jvm args and some default jvm args** are on the command line, therefore my application seems to crash. the instance shuts down and is restarted in an endless loop. I basically need most RAM of the instance off heap.

Jan

**Comment by axelfontaine** [comment]
This has now been implemented in 1.23.3.1189 

Now Boxfuse now doesn't add a default for -Xms anymore and the computed -Xmx value is overridden when explicitly specified in -jvm.args=...

**Comment by jtheuer** [comment]
Excellent, it works in the new version, thanks for fixing!
